src/XMLimport.cpp

```diff
@@ -1487,8 +1519,7 @@ void XMLimport::readTimerGroup( TTimer * pParent )
             }
             else if( name() == "script")
             {
-                QString script = readElementText();
-                pT->setScript( script );
+                tempScript = readElementText();
```

Comment:
Having revised the code for the TTrigger case it seems sensible to repeat the patter for all the other Mudlet item types (Action/Alias/Ket/Script/Timer) in the same manner.